app/src/main/java/com/example/footprnt/Map/Util/MapConstants.java

```diff
@@ -25,17 +22,18 @@
     public static int POST_RADIUS = 5;
 
     // Map Styles:
-    public static String map_style = "map_style";
-    public static int style_aubergine = R.raw.style_json_aubergine;
-    public static int style_darkmode = R.raw.style_json_darkmode;
-    public static int style_silver = R.raw.style_json_silver;
-    public static int style_retro = R.raw.style_json_retro;
-    public static int style_basic = R.raw.style_json_basic;
+    public static String MAP_STYLE = "map_style";
+    public static int STYLE_AUBERGINE = R.raw.style_json_aubergine;
+    public static int STYLE_DARKMODE = R.raw.style_json_darkmode;
+    public static int STYLE_SILVER = R.raw.style_json_silver;
+    public static int STYLE_RETRO = R.raw.style_json_retro;
+    public static int STYLE_BASIC = R.raw.style_json_basic;
+    public static BitmapDescriptor DEFAULT_MARKER = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_BLUE);
 
-    public static String menuItems[] = {"Create", "View", "Discover", "Current"};
+    // Filter menu items
+    public static String MENU_ITEMS[] = {"Create", "View", "Discover", "Current"};
```

Comment:
These menu items are something you want to put in string resources so that they can be translated easier in future.